<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertWidget</class>
 <widget class="QWidget" name="InsertWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>477</width>
     <height>156</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0">
     <widget class="QLabel" name="timeEndLabel">
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Time end</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="dateLabel">
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTimeEdit" name="timeEndEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="statusBox"/>
    </item>
    <item row="2" column="1">
     <widget class="QTimeEdit" name="timeStartEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="statusButton">
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Now</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="dateButton">
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Now</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="timeStartLabel">
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Time start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="plusLayout">
      <item>
       <widget class="QPushButton" name="plus60Button">
        <property name="text">
         <string>+60</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="plus45Button">
        <property name="text">
         <string>+45</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="plus30Button">
        <property name="text">
         <string>+30</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="plus15Button">
        <property name="text">
         <string>+15</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(35, 86, 31);</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
